src/chigame/users/views.py

```diff
@@ -52,25 +52,22 @@ def get_redirect_url(self):
 user_redirect_view = UserRedirectView.as_view()
 
 
-@login_required
-def user_list(request):
-    users = User.objects.all()
-    table = UserTable(users)
-    context = {"users": users, "table": table}
-
-    # Add information about top ranking users, total points collected, etc.
-
-    return render(request, "users/user_list.html", context)
-
-
 @login_required
 def user_detail(request):
```

Comment:
I think all of this functionality needs to be in user_profile_detail_view and not user_detail. The issue and PR description state that the list of friends will be displayed under the user's profile page, but right now it's under the url at user-detail. I would scrap this view entirely and place all the code in the user_profile_detail_view (with the additional check that the profile the user is currently on is their own).